tsdb/engine/tsm1/copy_or_link_unix.go

```diff
@@ -0,0 +1,17 @@
+//go:build !windows
+// +build !windows
+
+package tsm1
+
+import (
+	"fmt"
+	"os"
+)
+
+// copyOrLink - allow substitution of a file copy for a hard link when running on Windows systems.
+func copyOrLink(oldPath, newPath string) error {
```

Comment:
It might be more straightforward to just check `runtime.GOOS == "windows"`, but this is fine

Reply:
That was my original implementation, but reading various Go blogs convinced me conditional compilation was more idiomatic.